### DISCUSSION TITLE
Turn off rezoom on tour after adding destination/intermediate point?

### DISCUSSION THREAD

**Discussion by SomeMichael** [osmandapp/OsmAnd#14878] [Ideas]
Hi! After adding an additional destination or an intermediate point to a route Osmand does zoom onto the whole updated route. Is there a way to prevent this behaviour? Just do no zooming/moving/whatever at all…
This would be useful in my case for forcing the navigation onto specific roads/tracks. So in that case, I just want to see my changes 'locally', not having to search for the area where the change happened again. Maybe I have to add an intermediate point close to the point I just added…
Thanks!

**Comment by vshcherb** [comment]
I don't know such option so far.

**Reply by SomeMichael** [reply]
Ok, thanks! I move this topic to _ideas_.
